amethyst_ui/src/image.rs

```diff
@@ -1,12 +1,17 @@
 use amethyst_assets::Handle;
 use amethyst_core::ecs::{Component, DenseVecStorage};
-use amethyst_rendy::Texture;
+use amethyst_rendy::sprite::TextureCoordinates;
+use amethyst_rendy::{SpriteRender, Texture};
 
 /// Image used UI widgets, often as background.
 #[derive(Debug, Clone, PartialEq)]
 pub enum UiImage {
     /// An image backed by texture handle
     Texture(Handle<Texture>),
+    /// An image backed by partial texture
+    PartialTexture(Handle<Texture>, TextureCoordinates),
```

Comment:
`An image backed by a texture cropped to specified rectangle` would be more descriptive